CIbuild.sh

```diff
@@ -21,8 +21,19 @@ cd ..
 echo "Building..."
 make -j 2;
 make -j 2 test ARGS="-V";
+
+echo "Testing..."
 cd Server/;
 if [ "$TRAVIS_CUBERITE_BUILD_TYPE" != "COVERAGE" ]; then
-	echo restart | $CUBERITE_PATH;
-	echo stop | $CUBERITE_PATH;
+	$CUBERITE_PATH << EOF
+load APIDump
+apicheck
+restart
```

Comment:
For some reason the `restart` command seems to be ignored by the server. I don't care much at this moment, but we might want to look at it later.